scripts/changeset/get-full-changelog.mts

```diff
@@ -42,13 +45,14 @@ async function getChangelogInfo(octokit: Octokit, changeset: NewChangeset): Prom
     pull_number: prNo!,
   });
 
-  const prType = title.replace(/(\w+).*/, '$1'); // chore!: add something -> chore
+  const prType = title.replace(/(\w+).*/, "$1"); // chore!: add something -> chore
   const isBreaking = title.includes(`${prType}!`);
 
-  const titleDescription = title.replace(/\w+\W+(.*)/, '$1'); // chore!: add something -> add something
-  const summary = titleDescription.charAt(0).toUpperCase() + titleDescription.slice(1);
+  const titleDescription = title.replace(/\w+\W+(.*)/, "$1"); // chore!: add something -> add something
+  const summary =
+    titleDescription.charAt(0).toUpperCase() + titleDescription.slice(1);
 
-  const markdown = `- ${prLink}, ${summary}, by ${user}`;
+  const markdown = `- ${prLink} - ${summary}, by ${user}`;
```

Comment:
Replaced `,` by `-`.

From:
 - [#NNN](#), Pretty nice PR title, by @arboleya 

To:
 - [#NNN](#) - Pretty nice PR title, by @arboleya